File: test/fixtures/output/java/restclient/headers.java
RestClient restClient = RestClient.create();

ResponseEntity<String> response = restClient
  .method(HttpMethod.GET)
  .uri("http://mockbin.com/har")
  .header("accept", "application/json")
  .header("x-foo", "Bar")
  .header("quoted-value", "\"quoted\" 'string'")
  .retrieve()
  .toEntity(String.class);

File: test/fixtures/output/csharp/restsharp/headers.cs
var client = new RestClient("http://mockbin.com");
var request = new RestRequest("/har", Method.Get);
request.AddHeader("accept", "application/json");
request.AddHeader("x-foo", "Bar");
request.AddHeader("quoted-value", "\"quoted\" 'string'");
var response = client.Execute(request);